scripts/agent.yml

```diff
@@ -194,6 +194,10 @@ tasks:
           s3_copy_files:
               - {'source': {'path': 'evergreen/smoke/${build_id}-${build_variant}/evergreen-${task_name}-${revision}', 'bucket': 'mciuploads'},
                 'destination': {'path': 'evergreen/smoke/${build_id}-${build_variant}/evergreen-${task_name}-${revision}-copy', 'bucket': 'mciuploads'}}
+              - {'source': {'path': 'evergreen/smoke/${build_id}-${build_variant}/evergreen-${task_name}-${revision}-2', 'bucket': 'mciuploads'},
```

Comment:
Can the noop be the middle file rather than the last, to ensure that we continue past the noop?

Reply:
yes, I added another file at the end

Reply:
new patch with the changes: https://spruce.mongodb.com/version/61dee74d3627e037b1aaf66d/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC

Reply:
Alternatively instead of copying 3 full files, we could do <file 1>, <file 1>, <file 2> but i'm fine with either approach